features/fixtures/mazerunnerspring/src/main/java/com/bugsnag/mazerunnerspring/Scenario.java

```diff
@@ -0,0 +1,83 @@
+package com.bugsnag.mazerunnerspring;
+
+import com.bugsnag.Bugsnag;
+import com.bugsnag.delivery.Delivery;
+import com.bugsnag.serialization.Serializer;
+import org.apache.log4j.Logger;
+import org.springframework.util.StringUtils;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Calendar;
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+
+
+public abstract class Scenario {
+
+    private static final Logger LOGGER = Logger.getLogger(Scenario.class);
+
+    protected Bugsnag bugsnag;
+
+    public Scenario() {
+        // NOTE: this should already be configured by @Config
+        // all this does is get the instance
+        bugsnag = TestRestController.getBugsnag();
```

Comment:
We should pass this in as a constructor parameter instead.

Reply:
Changing this makes things more difficult with constructing the scenarios / passing the parameter to the base class, so I would prefer to leave this as it is